src/runtime/vulkan/vulkan_instance.cc

```diff
@@ -59,6 +59,14 @@ VulkanInstance::VulkanInstance() {
     std::vector<const char*> required_extensions{};
     std::vector<const char*> optional_extensions{"VK_KHR_get_physical_device_properties2"};
 
+    // Check if RGP support is needed. If needed, enable VK_EXT_debug_utils extension for
+    // inserting debug labels into the queue.
+    const char* val = getenv("TVM_USE_AMD_RGP");
```

Comment:
Use `BoolEnvironmentVar`
https://github.com/apache/tvm/blob/ef7143e1fc1a12993f584ce37dbde38b75966b40/src/runtime/vulkan/vulkan_instance.cc#L36

Reply:
> Use `BoolEnvironmentVar`
> 
> https://github.com/apache/tvm/blob/ef7143e1fc1a12993f584ce37dbde38b75966b40/src/runtime/vulkan/vulkan_instance.cc#L36

Hi @masahi,
Thanks for your time in reviewing the changes.  Will Implement your suggestion.
Thanks,
Anurag